.github/workflows/one-line-cr-bot.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
     # Get one-line-scan, the tool we will use for analysis
     - name: Get OneLineScan
-      run:  git clone -b one-line-cr-bot https://github.com/nmanthey/one-line-scan.git ../one-line-scan
+      run:  git clone -b one-line-cr-bot https://github.com/awslabs/one-line-scan.git ../one-line-scan
```

Comment:
From the one-line-cr-bot:

```
Run git clone -b one-line-cr-bot https://github.com/awslabs/one-line-scan.git ../one-line-scan
Cloning into '../one-line-scan'...
fatal: Remote branch one-line-cr-bot not found in upstream origin
Error: Process completed with exit code 128.
```

Reply:
Agreed, this should be changed. I will close this PR for now, and instead work on another one that comes with comments on PRs: #134 134